src/puppetlabs/trapperkeeper/services/webserver/jetty9_config.clj

```diff
@@ -323,14 +323,18 @@
              "Error: More than one default server specified in configuration"))))
 
 (defn selectors-count
-  "The number of selector threads that should be allocated per connector per core."
+  "The number of selector threads that should be allocated per connector per
+  core.  This algorithm duplicates the default that Jetty 9.2.7.v20150116 uses.
+  See: https://github.com/eclipse/jetty.project/blob/jetty-9.2.7.v20150116/jetty-server/src/main/java/org/eclipse/jetty/server/ServerConnector.java#L229"
   [num-cpus]
-  num-cpus)
+  (max 1 (min 4 (int (/ num-cpus 2)))))
```

Comment:
Did the Jetty defaults change since 9.1.0, or were we just not using the Jetty defaults for these?